web/pdf_page_view.js

```diff
@@ -163,6 +167,7 @@ class PDFPageView {
     this.maxCanvasPixels =
       options.maxCanvasPixels ?? AppOptions.get("maxCanvasPixels");
     this.pageColors = options.pageColors || null;
+    this.#enableHWA = !!options.enableHWA;
```

Comment:
For consistency with the other viewer components:
```suggestion
    this.#enableHWA = options.enableHWA || false;;
```